### PR TITLE
treehouses speedtest (fixes #433)

### LINKED ISSUE #433: treehouses speedtest?

### PR DESCRIPTION
**Pull request by vmnet8**
add new feature to test internet download and upload speed
fixs #433

### CODE REVIEW

File: modules/speedtest.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+function speedtest {
+  /usr/bin/speedtest "$@"
```
Comment (pattanawadee88) [inline modules/speedtest.sh]:
```
root@pattanawadee88:~/cli# ./cli.sh speedtest
/root/cli/modules/speedtest.sh: line 4: /usr/bin/speedtest: No such file or directory
```
we should check if the speedtest is installed

Comment (vmnet8) [reply modules/speedtest.sh]: you need to install speedtest-cli first, then try again
```
apt install speedtest-cli
```

**Comment by dogi** [comment]
```
$ which speedtest
/usr/bin/speedtest
```
could help you to check if speedtest is installed

File: modules/speedtest.sh (hunk 1 above)
Comment (vmnet8) [reply modules/speedtest.sh]: 110 image installed speedtest-cli, you can test right now on new image.

**Pull request merged**